src/command.ts

```diff
@@ -17,7 +17,7 @@ export interface RunCommandOptions {
 export async function runCommand<T extends ArgsDef = ArgsDef>(
   cmd: CommandDef<T>,
   opts: RunCommandOptions,
-): Promise<void> {
+): Promise<void | any> {
```

Comment:
For the forward compatibility, we might return `{ result }` object format

Reply:
@pi0 made the change you requested! Thanks!